src/amo/containers/DetailPage.js

```diff
@@ -1,37 +0,0 @@
-import React from 'react';
-import { compose } from 'redux';
-import { connect } from 'react-redux';
-
-import AddonDetail from 'amo/components/AddonDetail';
-import { UNKNOWN } from 'core/constants';
-import { loadAddonIfNeeded, safeAsyncConnect } from 'core/utils';
-
-
-export class DetailPageBase extends React.Component {
-  render() {
-    return (
-      <div className="DetailPage">
-        <AddonDetail {...this.props} />
-      </div>
-    );
-  }
-}
-
-export function mapStateToProps(state, ownProps) {
-  const { slug } = ownProps.params;
-  const addon = state.addons[slug];
-  const installation = state.installations[addon.guid] || { status: UNKNOWN };
-  return {
-    addon,
-    ...addon,
-    ...installation,
```

Comment:
As far as I could tell, only `status` was getting used here. The `isPreviewingTheme` and `themePreviewNode` were getting set by [`withInstallHelpers()`](https://github.com/kumar303/addons-frontend/blob/ef5669d0230e1a40746c14f0a2fc0da1ef765aff/src/core/installAddon.js#L101-L102).